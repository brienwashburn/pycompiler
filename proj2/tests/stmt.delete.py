del a
del a, b, c
del *a
del *a, b, c
del *a, *b, *c
del a,
del a, b, c,
del *a, *b, *c
del foo[bar]
del quux.bar
