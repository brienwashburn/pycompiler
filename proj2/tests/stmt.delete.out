(Module
 (Delete (Name a))
 (Delete (Name a) (Name b) (Name c))
 (Delete (Starred (Name a)))
 (Delete (Starred (Name a)) (Name b) (Name c))
 (Delete (Starred (Name a)) (Starred (Name b)) (Starred (Name c)))
 (Delete (Name a))
 (Delete (Name a) (Name b) (Name c))
 (Delete (Starred (Name a)) (Starred (Name b)) (Starred (Name c)))
 (Delete (Subscript (Name foo) (Index (Name bar))))
 (Delete (Attribute (Name quux) bar)))
